packages/xo-server/src/api/vm.js

```diff
@@ -12,6 +12,10 @@ import { forEach, map, mapFilter, parseSize } from '../utils'
 
 // ===================================================================
 
+export function getVmsHaValues () {
+  return ['Restart', 'Restart if possible', 'Disable']
```

Comment:
Those need to be the actual values expected by XenServer: "restart", "best-effort" and "".